gulp-tasks/test-integration.js

```diff
@@ -92,13 +103,13 @@ gulp.task('test-integration', async () => {
   }
 
   logHelper.log(`Downloading browsers......`);
-  await seleniumAssistant.downloadLocalBrowser('chrome', 'stable', 24);
-  await seleniumAssistant.downloadLocalBrowser('chrome', 'beta', 24);
-  await seleniumAssistant.downloadLocalBrowser('firefox', 'stable', 24);
-  await seleniumAssistant.downloadLocalBrowser('firefox', 'beta', 24);
+  const expiration = 24;
+  await seleniumAssistant.downloadLocalBrowser('chrome', 'stable', expiration);
```

Comment:
Why do one at a time? Maybe add a comment.

Reply:
It was previously one at a time. I tried doing them all at once in an interim commit as part of this PR, and it led to the following Travis CI run: https://travis-ci.org/GoogleChrome/workbox/builds/338757961#L3887

That shows that the Firefox tests never ran and Travis eventually timed out, so I just went back to the original sequential download.

I'll open a `selenium-assistant` issue to track whether there's any technical reason why simultaneous downloads might not work.

Reply:
See https://github.com/GoogleChromeLabs/selenium-assistant/issues/110